data/campaigns/Delfadors_Memoirs/scenarios/06_Swamps_of_Illuven.cfg

```diff
@@ -131,29 +133,70 @@
             side=1
             variable=fee
         [/store_gold]
-        {VARIABLE_OP fee multiply 8}
+        {VARIABLE_OP fee multiply {ON_DIFFICULTY 6 7 8}}
         {VARIABLE_OP fee divide 10}
         {VARIABLE_OP fee round 0}
+        {VARIABLE garrath_hostile yes}
+        {VARIABLE times_garrath_paid 0}
         [message]
             speaker=Garrath
             message=_"Greetings, strangers! This swamp is dangerous... You wanna cross it, you’ll need protection — cost you only $fee gold!"
             [option]
                 label=_"Thanks very much. Here’s the gold..."
                 [command]
-                    {VARIABLE_OP fee multiply -1}
                     [gold]
                         side=1
+                        amount=-$fee
+                    [/gold]
+                    # take his gold:
+                    [remove_item]
+                        x=4
+                        y=3
+                    [/remove_item]
+#ifdef EASY
+                    [message]
+                        speaker=Garrath
+                        message=_"You know, you really ought to be more careful before handing over money to suspicious strangers like me. But, since I'm feeling nice at the moment, I think I will actually keep my word and offer you some protection."
+                    [/message]
+                    [message]
+                        speaker=Harold
+                        message=_"Hey, what's your problem, Garrath? I thought we had a deal!"
+                    [/message]
+                    [message]
+                        speaker=Garrath
+                        message=_"Yeah, well, I changed my mind."
+                    [/message]
+                    [modify_side]
+                        side=2
+                        team_name=allies
+                    [/modify_side]
+                    {VARIABLE garrath_hostile no}
+                    {VARIABLE times_garrath_paid 1}
+                    [gold]
+                        side=2
                         amount=$fee
                     [/gold]
+                    # we are already inside an "ifdef EASY" block, so no need to vary this on difficulty:
+                    {CAPTURE_VILLAGES 2 4 3 4}
+                    [redraw]
+                    [/redraw]
+                    [message]
+                        speaker=Delfador
+                        message=_"...I'm not entirely sure what just happened here, but I guess I should be grateful?"
+                    [/message]
+                    [message]
+                        speaker=Lionel
+                        message=_"While it appears we lucked out this time, I fear for the travelers for whom this man does NOT keep his word..."
```

Comment:
I didn't realise this is a relatively recent change (I'm currently replaying DM in 1.14.17). I changed this to 'we were lucky' in the en_GB localisation, but if we're generally going for an archaic tone shouldn't we avoid the phrase 'lucked out'? To me it sounds like a modern Americanism - not sure if you have a different view on that.

Reply:
Yup, I agree. If you ever catch these, please feel free to open a PR about them.

Reply:
Thanks for confirming.